apps/ledger-live-desktop/src/newArch/Collectibles/utils/collectionUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { NFT, ProtoNFT } from "@ledgerhq/types-live";
+
+export const filterHiddenCollections = (
+  collections: (ProtoNFT | NFT)[] | Record<string, (ProtoNFT | NFT)[]>,
```

Comment:
maybe you can create a sub type for `ProtoNFT | NFT`

Reply:
Done ✅